packages/ai-jsx/src/react/core.ts

```diff
@@ -53,6 +54,14 @@ export function createElementSerializer(map: ComponentMap<any>): ElementSerializ
       );
     }
 
+    if (element.tag === Image) {
+      return {
```

Comment:
It's unfortunate that the current state of the React integration has three different ways AI.JSX elements manifest as React elements (corresponding to architectures 2, 3, and 4) but I think it's overwhelmingly preferable to have additional marginal complexity in places that are already complex (with the _same type of complexity_ IMO!) than spread the complexity elsewhere.